packages/superset-ui-preset-chart-xy/src/components/ChartFrame.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { isDefined } from '@superset-ui/core';
+
+function checkNumber(input: any): input is number {
+  return isDefined(input) && typeof input === 'number';
+}
+
+interface Props {
```

Comment:
`Props` should be a `type` not `interface`? 

looks like we have instances of `interface` in `superset-ui` but I thought we aligned on `type` because the styleguide recommends
> Use type for React components and constructs that can only be expressed with a type alias. Use interface for everything else. For a technical explanation of the differences and trade-offs, see [this post](https://medium.com/@martin_hotell/interface-vs-type-alias-in-typescript-2-7-2a8f1777af4c) about Interface vs Type alias in TypeScript 2.7.

Reply:
sgtm